src/features/swap/components/SwapForm.tsx

```diff
@@ -151,7 +151,15 @@ const SwapFormInput = ({
           inputMode="decimal"
         />
       }
-      infoChild={<BalanceTools mode={mode} onClickUseMax={setMaxAmount} balance={balance} />}
+      infoChild={
+        <div>
+          <BalanceTools mode={mode} onClickUseMax={setMaxAmount} balance={balance} />
+          <i className="text-[10px] text-color-secondary relative">
+            <span className="absolute left-[-0.5rem] top-0">*</span>
+            Only unlocked funds are shown above. For Locked funds see the Govern tab.
+          </i>
+        </div>
+      }
```

Comment:
should there also be a check for the mode here, similar to what's going on in BalanceTools? `mode === Mode.unstake`

Reply:
Woopsie doopsie. Great catch.